OpenSim/Analyses/JointReaction.h

```diff
@@ -72,12 +72,12 @@ OpenSim_DECLARE_CONCRETE_OBJECT(JointReaction, Analysis);
         /* Joint reference*/
         const Joint* joint;
         /* The body upon which the force is applied */
-        const PhysicalFrame* appliedOnBody;
+        const Frame* appliedOnBody;
```

Comment:
This change was necessary since `findBaseFrame()` returns a `Frame`. I didn't notice the comment, so that does make this look odd. As I understand from discussing this method yesterday, a "base frame" can only be a `PhysicalFrame` or a `Body`, so this problem might be about the return type of `findBaseFrame()`?